mta/mta.sw.yaml

```diff
@@ -126,7 +126,8 @@ states:
         actionDataFilter:
           toStateData: ".taskgroup"
     transition: poll
-  - actions:
+  - name: poll
```

Comment:
why MTA resources are modified in this PR?

Reply:
I just moved the name field on top of the section, so it's easier to understand the state name. I can rmeove it if you prefer (and open a critical issue for doing it later 😱 )

Reply:
no, it is fine. I was just surprised to see this here.
but a separate PR could be useful just to increase the chances for that part to be merged before the escalation workflow is (due to 2 different e2e tests and so.)